### ISSUE TITLE
Keybase not using User-Agent to scan website proofs; being blocked

### ISSUE THREAD

**Issue by stml** [keybase/client#8636]
According to one of my website hosts, Keybase does not send a User-Agent id when re-scanning for domain identities. See the server log below:

```
--------------------------------------------------------
54.84.133.185 - - [25/Sep/2017:07:21:55 +0100] "GET /keybase.txt HTTP/1.1" 200 2941 "-" "keybase-proofs/2.1.17"
54.84.133.185 - - [25/Sep/2017:08:45:42 +0100] "GET / HTTP/1.1" 404 - "-" "-"
54.84.133.185 - - [25/Sep/2017:09:14:34 +0100] "GET / HTTP/1.1" 404 - "-" "-"
--------------------------------------------------------
```

As a result, this host - and possibly others - block the Keybase IP as potentially malicious. Should this be fixed?

**Comment by maxtaco** [comment]
keybase-proofs/2.1.17

**Comment by stml** [comment]
Yes, that User-Agent is sent the first time, but not on subsequent checks, so the block in initiated...

**Comment by maxtaco** [comment]
I am not sure what that other request is.

But this seems like a ineffective security policy.  Attackers will just lie
about their user agent.

**Comment by stml** [comment]
Not here to defend this as a security policy, merely relating that this is this host's policy, and as a result I can't authenticate my domain.

**Comment by maxtaco** [comment]
I have a guess as to where this might be coming from, but I'm not sure.  I'll push something later today, hopefully, but that won't affect people who have written down (silly, in my opinion) blacklisting rules for our IP. Thanks for your feedback.
